core/src/main/java/io/ably/lib/realtime/RealtimeChannelBase.java

```diff
@@ -137,7 +141,7 @@ public void attach(CompletionListener listener) throws  AblyException {
         this.attach(false, listener);
     }
 
-    private void attach(boolean forceReattach, CompletionListener listener) {
+    void attach(boolean forceReattach, CompletionListener listener) {
```

Comment:
Why is this method public now? If it's a part of the API then we should add a proper javadoc to it (just like the other `attach` methods have)

Reply:
This method is needed internally in AblyRealtimeBase.reAttach(). We removed the visibility modifier so it is only visible in our package.

Reply:
True, my bad, this method is not public but visible only in the same package 😅